knative-operator/pkg/common/telemetry/telemetry_ucontroller.go

```diff
@@ -0,0 +1,100 @@
+package telemetry
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/runtime"
+	kafkasourcev1beta1 "knative.dev/eventing-contrib/kafka/source/pkg/apis/sources/v1beta1"
+	eventingsourcesv1beta1 "knative.dev/eventing/pkg/apis/sources/v1beta1"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var (
+	ServingObjects = []runtime.Object{
+		&servingv1.Service{},
+		&servingv1.Revision{},
+		&servingv1.Route{},
```

Comment:
Why no `Configuration`?

Reply:
I thought the same, in SRVKS mentions only the above, so from a telemetry perspective may not be useful. I can add it if it useful for telemetry.

Reply:
I will add it, should be of no harm.